static/scripts/homework.js

```diff
@@ -50,13 +30,6 @@ window.onload = function onload() {
 	}
 };
 
-function isSubmissionGradeUpload() {
-	// Uses the fact that the page can only ever contain one file upload form,
-	// either nested in the submission or the comment tab. And if it is in the
-	// comment tab, then it is the submission grade upload
-	return $('#comment .section-upload').length > 0;
-}
-
 function showAJAXError(req, textStatus, errorThrown) {
 	if (textStatus === 'timeout') {
 		$.showNotification($t('global.text.requestTimeout'), 'danger');
```

Comment:
Die Fehlermeldung in Zeile 64 mit 'homework.text.fileAlreadyExists' kannst du entfernen.